test/inductor/test_cudagraph_trees.py

```diff
@@ -4101,6 +4101,41 @@ def foo(x):
             compiled_out = compiled_foo(x)
             self.assertEqual(eager_out, compiled_out)
 
+        # Use autotune_at_compile_time=True to test standalone_compile
+        @parametrize("autotune_at_compile_time", [True])
```

Comment:
Can I ask what the use case is for autotune_at_compile_time ? hwo did this come up?